controllers/ControllerHome.php

```diff
@@ -18,25 +18,25 @@ public function __construct( $twig ) {
 		$this->_twig = $twig;
 		$this->_server = ( new ConstantGlobal(ServerRequest::fromGlobals()) )->getServerName()['SERVER_NAME'];
 
-		$this->renderHome();
-
 	}
 
-	public function renderHome(){
+	public function home(){
 		$posts = new PostsManager();
 		$allPosts = $posts->listPosts();
 		$key = (new Session)->getKey('user');
 
+
 		if(!empty($key)){
 			echo $this->_twig->render( 'home.twig', ['allPosts' => $allPosts,
 			                                         'logged'=> TRUE,
 			                                         'user'=> $key['pseudo'],
-													  'server' => $this->_server]);
+			                                         'admin' => $key['admin'],
```

Comment:
Codacy found an issue: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$key['admin']'.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=6061935)